matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/model/LiveLocationBeaconContent.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.android.sdk.api.session.room.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import org.matrix.android.sdk.api.session.room.model.message.LocationAsset
+
+@JsonClass(generateAdapter = true)
+data class LiveLocationBeaconContent(
+        /**
+         * Indicates user's intent to share ephemeral location.
+         */
+        @Json(name = "m.beacon_info") val beaconInfo: BeaconInfo? = null,
+        /**
+         * Beacon creation timestamp.
```

Comment:
I'm not sure if we do it in other places, but it might be handy to know the timestamp format, for example epoch seconds or milliseconds

Reply:
Renamed and used clock class.